modules/data.atmosphere/R/solar_angle.R

```diff
@@ -34,7 +34,8 @@ cos_solar_zenith_angle <- function(doy, lat, lon, dt, hr) {
 #' @return `numeric(1)` length of the solar day, in hours.
 #' @export
 equation_of_time <- function(doy) {
-  stopifnot(doy <= 366)
+  stopifnot(doy <= 367) #changed from 366 to 367 to account for leap years
```

Comment:
No need to change this time, but for future reference bug fixes like this that are better done as their own PRs rather than rolling them in with new features. Putting them in a separate PR makes it
* easier to make sure the right people see them
* faster to get the bug fix reviewed and merged (it doesn't have to wait for the feature PR to be complete)
* faster to get the feature reviewed and merge (fewer files to read and see track of!)
* Lets other people start using the fixed code ASAP

Reply:
Yes thanks for the advice. I made this fix back in Feb of this year when we were having issues getting PR approved because of the checks failing so it got rolled into this PR. In the future I absolutely plan on submitting smaller PR.